### PR TITLE
Feature devrel#251

### PR DESCRIPTION
**Pull request by VEG-Ginna**
Notes:

- Line 47 - "This guide" - do not use this construction. Explicitly state the guide's name and add a link. Always follow the word "this" with an identifier.
- Line 191 - Add information or a link to the doc which tells you how to determine success.

### CODE REVIEW

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md [hunk 1]
```diff
@@ -2,153 +2,150 @@
 title: DUNE
 ---
 
-[Docker Utilities for Node Execution (DUNE)](https://github.com/AntelopeIO/DUNE) is a client tool that allows blockchain developers and node operators to perform boilerplate tasks related to smart contract development and node management functions.
+[Docker Utilities for Node Execution (DUNE)](https://github.com/AntelopeIO/DUNE) is a client tool. 
+As a blockchain developer or node operator, use DUNE for your common tasks.
 
-Before getting started with smart contract development, you need to learn about DUNE and how to install it on your platform.
+Before you begin, learn about DUNE and how to install it on your platform.
```
Comment (nsjames) [inline docs/20_smart-contracts/10_getting-started/10_dune-guide.md:8]: This changes the meaning. 

The intention here seemed to be "before you begin **smart contract development**.
Now it is ambiguous

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md [hunk 2]
```diff
@@ -2,153 +2,150 @@
...
 
 ### Installation
 
-DUNE can be installed and run on the following platforms:
+You can install and run DUNE n the following platforms:
```
Comment (nsjames) [inline docs/20_smart-contracts/10_getting-started/10_dune-guide.md:12]: Typo here "Dune n"

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md [hunk 3]
```diff
@@ -2,153 +2,150 @@
...
 * Linux
 * Windows
 * MacOS
 
-Installation instructions for each supported platform are available on the [DUNE's github project](https://github.com/AntelopeIO/DUNE) page.
+Each platform has their own installation instructions. Click [DUNE's GitHub project](https://github.com/AntelopeIO/DUNE) to view the instructions.
```
Comment (nsjames) [inline docs/20_smart-contracts/10_getting-started/10_dune-guide.md:17]: Simpler is better: 

View the [DUNE GitHub](https://github.com/AntelopeIO/DUNE) for installation instructions.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md [hunk 4]
```diff
@@ -2,153 +2,150 @@
...
 
 ```shell
 dune --import-dev-key <PRIVATE_KEY>
 ```
 
-## Node Management
+## Node management
 
-Creating a new local EOS blockchain is easy with DUNE.
+DUNE easily creates a new local EOS blockchain.
```
Comment (nsjames) [inline docs/20_smart-contracts/10_getting-started/10_dune-guide.md:33]: Please don't add verbosity to guides. 

They should be as short and succinct as possible to not waste a developer's time with unnecessary reading.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md [hunk 5]
```diff
@@ -2,153 +2,150 @@
...
+
+The following command creates a new node called `NODE_NAME`.  This node initiates with default settings. These default settings configure the node to serve as an API/producer node. You can deploy smart contracts to it and perform tests on it.
```
Comment (nsjames) [inline docs/20_smart-contracts/10_getting-started/10_dune-guide.md:35]: This flowed better with the explanation text below the code block. 

The commands/code should always be the primary, and the explanation of the code is the backup just in case the reader didn't understand what's happening from the code line itself. We get to the code as quickly as possible.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md [hunk 6]
```diff
@@ -2,153 +2,150 @@
...
 
 ```shell
 dune --start <NODE_NAME>
 ```
 
-The command above creates a new node called `NODE_NAME` and starts it with default settings. 
-The node is configured to serve as an API/producer node that you can deploy smart contracts to, and perform tests on.
-
-> ❔ **Errors**
+> 📝 **In Case of Errors**
```
Comment (nsjames) [inline docs/20_smart-contracts/10_getting-started/10_dune-guide.md:41]: Unnecessary change, and please don't use title case outside of h1 tags (`#`) unless it's a proper noun. 

If you see title case in these admonitions please correct them.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md [hunk 7]
```diff
@@ -2,153 +2,150 @@
...
 
-To shut down your node:
+The following command shuts down your node:
 
 ```shell
 dune --stop <NODE_NAME>
 ```
 
-To remove a node entirely:
+The following command removes a node:
```
Comment (nsjames) [inline docs/20_smart-contracts/10_getting-started/10_dune-guide.md:65]: It was already understood that the command is what causes the action. 

We don't need to write `The following command` 4 times. It is redundant and wastes the developer's time.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md [hunk 8]
```diff
@@ -2,153 +2,150 @@
...
 
 ```shell
 dune --bootstrap-system-full
 ```
 
-
 ## Account management
 
-You deploy contracts on top of accounts, and also use them to interact with your smart contracts. 
+You deploy contracts on top of accounts. Use accounts to interact with your smart contracts.
```
Comment (nsjames) [inline docs/20_smart-contracts/10_getting-started/10_dune-guide.md:86]: This changes the meaning, please revert.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md [hunk 9]
```diff
@@ -2,153 +2,150 @@
...
 ```
 
-
 ## Account management
 
-You deploy contracts on top of accounts, and also use them to interact with your smart contracts. 
+You deploy contracts on top of accounts. Use accounts to interact with your smart contracts. 
 
-To create a new account:
+The following command creates a new account:
```
Comment (nsjames) [inline docs/20_smart-contracts/10_getting-started/10_dune-guide.md:88]: Same issue as above, `the following command` makes the guide harder to read, not easier.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md [hunk 10]
```diff
@@ -2,153 +2,150 @@
...
-To get account info:
+The following command gets account info:
 
 ```shell
 dune -- cleos get account <ACCOUNT_NAME>
 ```
+## Smart contract development
 
-## Smart Contract Development
+Create a sample project to learn how to use DUNE to compile, deploy, and interact with smart contracts.
```
Comment (nsjames) [inline docs/20_smart-contracts/10_getting-started/10_dune-guide.md:101]: Our preference should always be first-person plural, or second-person point of view in guides. 

This is imperative point of view.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md [hunk 11]
```diff
@@ -2,153 +2,150 @@
...
 
 ```cpp
-#include <eosio/eosio.hpp>
+#include <eosio/eosio.hpp> 
 using namespace eosio;
 
-CONTRACT hello : public contract {
-   public:
-      using contract::contract;
+CONTRACT hello : public contract {ompilinc
```
Comment (nsjames) [inline docs/20_smart-contracts/10_getting-started/10_dune-guide.md:118]: Not sure what happened here, but please don't touch code blocks. 

Indentation changed, and `ompilinc` was added here

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md [hunk 12]
```diff
@@ -2,153 +2,150 @@
...
-
 ### Compile the contract
 
 From the root of your project, run the following command to compile your contract:
 
 ```shell
 dune --cmake-build .
 ```
-You will see your contract being compiled. If there are any errors you will see them in the output.
+You see your contract compiling. If there are any errors, you see them in the output.
```
Comment (nsjames) [inline docs/20_smart-contracts/10_getting-started/10_dune-guide.md:144]: Something about this sentence is weird, and hard to read. Perhaps it's the change in tense.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md [hunk 13]
```diff
@@ -2,153 +2,150 @@
...
 
 ### Deploy your contract
 
-We need to create an account for your contract, and then we can deploy it.
+Use the following commands to create an account and deploy the contract to the account.
```
Comment (nsjames) [inline docs/20_smart-contracts/10_getting-started/10_dune-guide.md:148]: First-person plural, or second-person. 

This sterilizes the content and makes it commands instead of a journey with the instructor.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md [hunk 14]
```diff
@@ -162,10 +159,12 @@ dune --deploy ./build/hello hello
...
 
-To interact with your contract you will send a transaction on your local EOS node. Transactions on EOS are made up of 
-`actions`, so we will send a single action to your contract.
+Transactions on EOS consist of 
+`actions`. To interact with your contract send a single action to your contract.
+
+You also need to create a test account from which to send the action.
 
-We will also create a test account to send the action from.
+Use the following commands to create a test account and send an action:
```
Comment (nsjames) [inline docs/20_smart-contracts/10_getting-started/10_dune-guide.md:167]:
```
Transactions on EOS consist of actions. To interact with your contract send a single action to your contract.

You also need to create a test account from which to send the action.

Use the following commands to create a test account and send an action:
```

This is still very verbose, and hard to read. `consist` is also a 12th-grade reading level, we are aiming lower than that to help non-English speakers.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md [hunk 15]
```diff
@@ -174,27 +173,31 @@ dune --create-account testaccount
...
 
-You should see a transaction executed successfully on the first time, and if you try to repeat this command it will 
-fail because that row already exists in the contract's database.
+You should see a transaction executed successfully. If you try to repeat this command, the command 
+fails because the row already exists in the contract's database.
 
 ### Get data from your contract
 
-You just added a row to the contract's database, let's fetch that data from the chain:
+You added a row to the contract's database.
```
Comment (nsjames) [inline docs/20_smart-contracts/10_getting-started/10_dune-guide.md:181]: Again, sterilization.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md [hunk 16]
```diff
@@ -174,27 +173,31 @@ dune --create-account testaccount
...
+
+Use the following command to fetch that data from the chain:
 
 ```shell
 # format: dune --get-table <CONTRACT> <SCOPE> <TABLE>
 dune --get-table hello hello users
 ```
 
-You should get a table result with one or more row. If you didn't make sure your interaction above was successful.
+You should get a table result with one or more rows. If you didn't, make sure your interaction above was successful.
```
Comment (nsjames) [inline docs/20_smart-contracts/10_getting-started/10_dune-guide.md:190]: `rows` was correct.

**Comment by nsjames** [comment]
`"This guide"` is fine when referring to... this guide.

**Comment by nsjames** [comment]
> Add information or a link to the doc which tells you how to determine success.

When they run 
```
dune --send-action hello test '[bob]' testaccount
```

It will output a clear error if it was unsuccessful

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 10 above)
Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:101]: This construction is second-person point of view. It is called "implied". "Implied subjects occur when a sentence does not state the doer of the action, but it is clear to whom the sentence is referring." This construction is a standard practice for instructions.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 11 above)
Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:118]: I primarily work with text. What needs to be corrected?

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 12 above)
Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:144]: The two sentences are passive voice. The rules are to take out the helping verb and to not use gerunds (-ing words).

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 14 above)
Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:167]: The first line of the example is a definition, typically written "A transaction on EOS is ..."  I could write this sentence as "A transaction on EOS is ... and contains actions.

The red highlighted sentence is passive due to the helping verb and ends in a preposition.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 13 above)
Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:148]: This construction is second-person point of view. It is called "implied". "Implied subjects occur when a sentence does not state the doer of the action, but it is clear to whom the sentence is referring." This construction is a standard practice for instructions.

Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:148]: The journey includes entering commands. Plus it is not done by more than one person (the reader). It's not good practice to imply others are involved with the action without explicitly identifying them.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 1 above)
Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:8]: "Before getting started ..." is Grade 12 level. The new construction is Grade 6.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 2 above)
Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:12]: Done

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 3 above)
Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:17]: Done

Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:17]: Done

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 6 above)
Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:41]: Got it

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 7 above)
Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:65]: Done

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 8 above)
Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:86]: These changes were done to simplify the language for Hemingway.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 9 above)
Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:88]: Done

Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:88]: Done

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 15 above)
Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:181]: Simplify language for Hemingway grade level.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 8 above)
Comment (nsjames) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:86]: Hemingway is just a tool, it does not mean aligning with it allows tw review to change the meaning of technical documents.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 11 above)
Comment (nsjames) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:118]: All of the indentation of the code block was messed up and there is an added `ompilinc` text there which breaks the code.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md [hunk 17]
```diff
@@ -2,55 +2,55 @@
...
 dune --import-dev-key <PRIVATE_KEY>
 ```
 
-## Node Management
+## Node management
 
-Creating a new local EOS blockchain is easy with DUNE.
+Use DUNE to easily create a new local EOS blockchain.
+
+The above command creates a new node called `NODE_NAME`.  This node initiates with default settings. These default settings configure the node to serve as an API/producer node. You can deploy smart contracts to it and perform tests on it.
```
Comment (nsjames) [inline docs/20_smart-contracts/10_getting-started/10_dune-guide.md:35]: Please revert the changes here as per this comment: https://github.com/eosnetworkfoundation/docs/pull/118#discussion_r1192244820

This now says the command above for the command below.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 1 above)
Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:8]: Done

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 8 above)
Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:86]: I am using Hemingway to meet the requirement for 5th grade level. Therefore most sentences will be shortened or divided into two or more sentences. This type of change is not meant to change the meaning, but to clarify the meaning of the information in smaller chunks in a simpler structure. That structure is noun. verb, object. It does not leave much space for nuance. 

Using the term "them" needs to be explicit - do you mean contracts or accounts?

Can I use the following? "You can deploy contracts on top of accounts. An account with a contract on it is no different from an account you use to interact with smart contracts." - that makes more sense to me.

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 11 above)
Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:118]: Done

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 12 above)
Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md:144]: The reason for the change is to remove passive voice.

Developer writing is typically full of passive voice because of your exact statement - "They either will be putting the command in the future, or have already put the command in the past. It's very unlikely they will read this the same second they are clicking enter on the command."

Removing passive voice is tricky. The structure of "being compiled" infers and action going on in the background. "Is compiled" and "you will see them" is definitely passive and is understandable for developers. Whereas, for translation passive voice can get confusing.

I prefer the Past: "Your contract is now compiled. If there were any errors you see them in the output."

File: docs/20_smart-contracts/10_getting-started/10_dune-guide.md (hunk 14 above)
Comment (VEG-Ginna) [reply docs/20_smart-contracts/10_getting-started/10_dune-guide.md]: Done

**Comment by nsjames** [comment]
This is too verbose of a PR. 

As we discussed, these will be done in phases from now on to reduce the amount of review required on the original writer.
